docs/README.md

```diff
@@ -126,14 +126,14 @@ You can exclude a heading from the table of contents:
 
 	{% top %}
 
-This will be replaced by a default back to top link. It is recommended to use these links at least at the end of each level-2 section.
+This will be replaced by default back to top link. It is recommended to use these links at least at the end of each level-2 section.
```

Comment:
I'm not really convinced. It still sounds wrong, and it strikes be as odd to remove it here despite being present for labels. I will revert this change (actually adjust it slightly to remove `default` because I have no idea what that is supposed to mean).